src/tv2_afvd_showstyle/config-manifests.ts

```diff
@@ -262,6 +262,46 @@ export const schemaConfigManifest: ConfigManifestEntry[] = [
 	}
 ]
 
+export const gfxDesignTemplates: ConfigManifestEntry[] = [
+	{
+		id: 'GFXDesignTemplates',
+		name: 'GFX Design Templates',
+		description: '',
+		type: ConfigManifestEntryType.TABLE,
+		required: true,
+		defaultVal: [],
```

Comment:
We still have the defaults for the designs in the `src\tv2-common\migrations\graphic-defaults.ts` file, which are inserted into the GFX Templates table with all the other templates [here](https://github.com/tv2/sofie-blueprints-inews/blob/fb594f25c0d945f09655087c092b93b91e2d26eb/src/tv2_afvd_showstyle/config-manifests.ts#L342). Shouldn't they be separated and inserted as the default here instead?

Reply:
I guess the migrations take care of it anyway, though it wasn't part of the task i assume this would be fairly easy to do if you want me to?